api/mysagw/case/serializers.py

```diff
@@ -72,3 +73,36 @@ def create(self, validated_data):
         )
 
         return instance
+
+
+class CaseTransferSerializer(drf_serializers.Serializer):
+    removable_assignees = drf_serializers.ListField(
+        child=drf_serializers.PrimaryKeyRelatedField(queryset=Identity.objects.all())
+    )
```

Comment:
Hmm, naming... Should this maybe be named `to_remove_assignees` instead?

Reply:
Done :+1: